### PR TITLE
Support multiple sim slots.

### PR DESCRIPTION
**Pull request by andremarcais**
Allow using a sim other than the default or the one in slot zero for sending sms messages.

See the related [pull request](https://github.com/termux/termux-api-package/pull/123) for termux-api-package.

### CODE REVIEW

File: app/build.gradle
Comment (xeffyr) [inline app/build.gradle:4]: Don't touch **SDK version**. Termux is not compatible with anything beyond 28.

**Review by xeffyr — CHANGES_REQUESTED** [review]

File: app/build.gradle
Comment (andremarcais) [reply app/build.gradle:4]: I'll see if it can be done in version 28, but at a glance it seems like all the sim slot stuff is API version 29. I bumped the version up because [``getSubscriptionIds``](https://developer.android.com/reference/android/telephony/SubscriptionManager#getSubscriptionIds(int)), which is used to setup the ``SmsManager`` for a given slot, was added in API level 29.

**Comment by Grimler91** [comment]
Any particular reason for failing silently instead of printing "error sim a lot not found" or similar when slot does not exist?

**Comment by andremarcais** [comment]
No, no particular reason. I just wasn't able to figure out how to do it quickly, so I omitted the feature in the interest of time since I didn't need it for what I was doing.

**Comment by andremarcais** [comment]
I can implement it some time soon.  Is putting a json status on stdout like the other commands ok? (It might break scripts that expect ``termux-sms-send`` to put nothing on stdout. Is that a concern?)

**Comment by Grimler91** [comment]
@andremarcais I guess there are no clear policies on how to handle problems in termux-api, some other commands log to syslog and fail silently as well. 

I think it would be good to return a (json) error message if sim card is not found, so that the user can immediately tell that the text message was not sent. We can also merge this for now (if you do not have time to work on it now) and try to improve the error handling of multiple api commands at a later stage

**Comment by andremarcais** [comment]
@Grimler91 Yeah, if this is good enough to be merged as is then go ahead. I'm probably not going to get to implement the json error message anytime soon.

Plus it probably makes sense to implement all of the error handling stuff latter. That subject probably deserves its own discussion.

**Comment by Grimler91** [comment]
Will wait a bit with releasing as it seems termux-sms-list prints in the reverse order after merging https://github.com/termux/termux-api/pull/407.

**Pull request merged**
